src/neo/neoSchema.js

```diff
@@ -0,0 +1,47 @@
+export const NeoTransactonOnline = {
```

Comment:
Typo in the variable name is causing tests to fail. Should be 'NeoTransactionOnline'

Reply:
Let's move to #154